gradle/spotbugs-exclude.xml

```diff
@@ -40,6 +40,12 @@ For a detailed description of spotbugs bug categories, see https://spotbugs.read
         <Bug pattern="DM_EXIT"/>
     </Match>
 
+    <Match>
+        <!-- Disable warnings about doProvileged, This has been deprecated
+            as of JDK 17 for removal, see https://openjdk.org/jeps/411 -->
+        <Bug pattern="DP_CREATE_CLASSLOADER_INSIDE_DO_PRIVILEGED"/>
```

Comment:
This comment confused me at first. Can you include the description of `DP_CREATE_CLASSLOADER_INSIDE_DO_PRIVILEGED` to make it clear to the reader?

> Classloaders should only be created inside doPrivileged block